scripts/templates/bookmark.handlebars

```diff
@@ -0,0 +1,17 @@
+<li
```

Comment:
File name: I think the convention is .hbs file extension now? I haven´t tested but its mostly about syntax highlighting in Atom/whatever IDE